tests/experimental/benchmark_pt2.py

```diff
@@ -0,0 +1,64 @@
+# flake8: noqa
+
+
+def run_benchmark(irreps_type):
+    import torch
+    from torch._inductor.utils import print_performance
+
+    # Borrowed from https://github.com/pytorch-labs/gpt-fast/blob/db7b273ab86b75358bd3b014f1f022a19aba4797/generate.py#L16-L18
+    torch.set_float32_matmul_precision("high")
+    import torch._dynamo.config
+    import torch._inductor.config
+
+    torch._inductor.config.coordinate_descent_tuning = True
+    torch._inductor.config.triton.unique_kernel_names = True
+
+    device = "cuda"
+    compile_mode = "max-autotune"  # Use reduce-overhead for quick compile times
+
+    from e3nn import o2, o3, experimental
```

Comment:
Where is o2 defined, did I missed it?

Reply:
Sorry for that was a bleedover from the other PR.